app/controllers/accounts_controller.rb

```diff
@@ -61,4 +62,8 @@ def ability_resource
     @account
   end
 
+  def administered_order_details
+    @administered_order_details ||= current_user.administered_order_details
```

Comment:
Does this get used more than once? 1) I'm wondering if it's necessary to extract out into its own method, and 2) since I believe it's returning an ActiveRecord::Relation, is the memoization necessary?